README.md

```diff
@@ -143,7 +144,61 @@ To remove, run:
 
     $ chmod +x buku.py
     $ ./buku.py
-
+#### Install server side
```

Comment:
Add a newline before the section.

Section name should be #### Install server

Reply:
I think the rest of the content should go to the README within the `server` directory. Also please link to jarun/Buku as server is finally going to be merged to the main repo.

Reply:
@rachmadaniHaryono when the review changes are done, can you please add a README within server and add the following contents there?

Reply:
There are places where we need to change the words or sentences. I think I will do that myself when the complete PR with all changes are done.

@tom-i I think a better approach would be to raise the PR against @rachmadaniHaryono's repo for now. Add the changes to the README within the server directory.

Reply:
So every notes I've made I should put to the README in server directory?
You've merged all the @rachmadaniHaryono changes? Because I would like to test it, if that changes works on normal repo.

Reply:
Imo just put it up here for review. 

 if everything is done and feature is ready to be merged I can merge tom-i branch myself and we can close the pr on main repo

Reply:
:+1:

Reply:
So I've made some small changes, but guys.. there is a problem, because I don't know how to push to the main Buku repo :D 
I'm newbie in GIT